capabilities_matrix/_topics/cloud_providers.md

```diff
@@ -2,60 +2,60 @@
+| Capacity & Utilization                     | Yes                       | Yes             | Yes                         | Yes                        | No                                    | No                                |
+| Capture Cloud Layer Timelines              | No                        | No              | No                          | Yes                        | Yes                                   | No                                |
+| Capture Instance Event Timelines           | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | No                                |
+| Discovery - Provider                       | No                        | No              | No                          | No                         | No                                    | No                                |
+| Disk Addition to Instance                  | Yes                       | Yes             | No                          | Yes                        | No                                    | Yes                               |
+| Key Pairs Inventory                        | Yes                       | No              | Yes                         | Yes                        | Yes                                   | Yes                               |
+| Key Pairs Management                       | Yes                       | No              | No                          | Yes                        | Yes                                   | Yes                               |
+| Reporting                                  | Yes                       | Yes             | Yes                         | Yes                        | No                                    | No                                |
+| Right Sizing                               | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | No                                |
+| Chargeback                                 | Yes                       | Yes             | Yes                         | Yes                        | No                                    | Yes                               |
+| Remote Console Instance Access             | No                        | No              | No                          | Yes                        | Yes (for NovaLink-managed hosts only) | No                                |
+| Snapshot Creation and Removal              | No                        | No              | No                          | Yes                        | Yes                                   | No                                |
+| Instance Compliance Enforcement            | Yes                       | Yes             | Yes                         | Yes                        | No                                    | No                                |
```

Comment:
Would this "just work"?

Reply:
(Compliance enforcement)

Reply:
Here's the documentation I read to understand this capability: https://www.manageiq.org/docs/reference/latest/policies_and_profiles_guide/index.html

Things like enforcing an sshd configuration inside the VM would require direct access to the VM OS (maybe there's a way to add a MIQ ssh key or something?).

I was also looking at the 'IBM PowerVC via the OpenStack API' column, which is marked `No`, and assuming there's more to this capability than what we've implemented.

Guidance from the MIQ team would be helpful here.

Reply:
Sounds good, agree with "No" for now.